core/src/main/java/tc/oc/pgm/api/party/Party.java

```diff
@@ -132,4 +135,20 @@ default boolean isParticipating() {
   default boolean isObserving() {
     return !this.isParticipating();
   }
+
+  @Override
+  @Nullable
+  default Match filterableParent() {
```

Comment:
Let's stick with the `get` style in this repo, that style is a remanent of the previous PGM.